index.bs

```diff
@@ -3096,6 +3493,20 @@ partial interface MLGraphBuilder {
                           [EnforceRange] unsigned long hiddenSize,
                           optional MLGruOptions options = {});
 };
+
+dictionary MLGruSupportLimits {
+  MLSupportLimits input;
+  MLSupportLimits weight;
+  MLSupportLimits recurrentWeight;
+  MLSupportLimits bias;
+  MLSupportLimits recurrentBias;
+  MLSupportLimits initialHiddenState;
+  MLSupportLimits output;
```

Comment:
Nit: gru returns a sequence of operands, naming it outputs?
```suggestion
  MLSupportLimits outputs;
```

Reply:
+1 Didn't notice that.